crates/rsonpath-lib/src/query/automaton/minimizer.rs

```diff
@@ -5,16 +5,18 @@
 use super::nfa::{self, NfaState, NfaStateId};
 use super::small_set::{SmallSet, SmallSet256};
 use super::state::StateAttributesBuilder;
-use super::{Automaton, NondeterministicAutomaton, State as DfaStateId, StateTable};
-use super::{Label, StateAttributes};
+use super::StateAttributes;
+use super::{
+    Automaton, NondeterministicAutomaton, State as DfaStateId, StateTable, TransitionLabel,
+};
```

Comment:
Please merge these.

Reply:
Done.  Is there possible interest in setting the following for rustfmt?

```
imports_granularity = "Crate"
group_imports = "One"
imports_layout = "Vertical"
```